packages/react/src/components/AppShell/app-shell.scss

```diff
@@ -33,16 +34,29 @@
     flex-grow: 1;
     transition: all 0.5s;
 
+    .oxygen-app-shell-navigation-wrapper {
+      background: var(--oxygen-palette-customComponents-Navbar-background);
+
+      .MuiDrawer-paper {
+        top: var(--oxygen-customComponents-AppShell-properties-navBarTopPosition);
+        bottom: 0;
+        height: auto;
```

Comment:
Why do we add these styles again in the app side? wondering since we already have them in the component in Oxygen UI.